### PR TITLE
SYS-781 - Create file upload form in django

### PR DESCRIPTION
**Pull request by darrowcoucla**
Connected to [SYS-781](https://jira.library.ucla.edu/browse/SYS-781)

A page was added that can be called from the legacy DLCS app. The page then calls the Oral History media processing script.
Per @kjallen , only the _path/filename_ and _ARK_ need to be sent to the script that he is working on.

-----
**Acceptance criteria:**

- [x] A Django page exists which receives data from another page ([SYS-780](https://jira.library.ucla.edu/browse/SYS-780))
- [x] This page retrieves basic item data from the database
        - Title
        - Author
        - Ark
- [x] This page displays a form supporting file upload requirements
        - File group
        - File picker which selects one file from hard-coded location
        - Files will not actually be uploaded
        - The processing script needs only path to file, not the actual file, from this form
- [x] Submitting the form sends data to a Django management command (TBD)

-----
**Typical Screenshots**

image

-----
**Changed Files**
```
README.md
docker_scripts/entrypoint.sh
oral_history/
    admin.py
oral_history/
    forms.py
oral_history/
    management/
        commands/
            run_script.py
    models.py
    scripts/
        process.py
    templates/
        oral_history/
            base.html
            files.html
            fileupload.html
            table.html
    urls.py
    views.py
```

### CODE REVIEW

**Comment by kjallen** [comment]
This looks like it has all the pieces I need, thank you for adding `file_group`, I overlooked that. There is a way to generate the dropdown with only valid `file_group` values from the DB but is not needed for now

File: oral_history/views.py [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
+
+    return render(request, 'oral_history/fileupload.html', {'form': form, 'query_results': query_results})
+
+
 def projects_table(request):
     query_results = ProjectItems.objects.all()
     return render(request, 'oral_history/table.html', {'query_results': query_results})
+
+
+def projects_files(request):
```
Comment (akohler) [inline oral_history/views.py:45]: @darrowcoucla Where is this view used?  I see the URL `/files` defined (and it works), the template displays info, but this seems like an unused partial duplicate of the full-featured file upload view/template.

File: oral_history/scripts/process.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+
+class Process():
+
+    def __init__(self, reports_dir):
+        print("Processing media file ...")
+
+    def run(self, file_group, file_name, item_ark):
+        print("script('"+str(file_group)+"', '" +
```
Comment (akohler) [inline oral_history/scripts/process.py:10]: @darrowcoucla A cleaner way to do this is with python's "f-strings" - see https://realpython.com/python-f-strings/ for examples.

Something like (untested) ```print(f'script({str(file_group)})') etc.```
Also, I think these 3 variables are all strings already, so no need to stringify them with `str()`.

File: oral_history/models.py [hunk 3]
```diff
@@ -2,17 +2,21 @@
 
 # Create your models here.
 
+
 class DlObjects(models.Model):
     objectid_pk = models.AutoField(primary_key=True, auto_created=True)
-    projectid_fk = models.ForeignKey('Projects', models.CASCADE, db_column='projectid_fk')
-    object_typeid_fk = models.ForeignKey('DlObjectTypes', models.CASCADE, db_column='object_typeid_fk')
+    projectid_fk = models.ForeignKey(
```
Comment (akohler) [inline oral_history/models.py:8]: @darrowcoucla I think all these changes in `models.py` are formatting done by your IDE, correct?  No actual model changes?  No action needed, just a lot of changes to read through.

`TODO` for me: Add python linting / formatting modules to this project so code is handled consistently, regardless of IDE / developer preferences.

**Review by akohler — CHANGES_REQUESTED** [review]

File: oral_history/views.py (hunk 1 above)
Comment (darrowcoucla) [reply oral_history/views.py:45]: Deprecated view and related code/file removed.

File: oral_history/scripts/process.py (hunk 2 above)
Comment (darrowcoucla) [reply oral_history/scripts/process.py:10]: f-strings inserted

File: oral_history/models.py (hunk 3 above)
Comment (darrowcoucla) [reply oral_history/models.py:8]: Yes, these formatting changes were made by VS Code. Sorry, I should have made a note. Linting sounds like a good idea.

**Review by akohler — APPROVED** [review]

**Pull request merged**